test/stdlib/Integers.swift.gyb

```diff
@@ -357,6 +371,18 @@ tests.test("${Type}/SubtractInPlace/Underflow") {
   var x = ${Type}.min
   f(&x)
 }
+
+tests.test("${Type}/SubtractInPlace/MaskingUnderflow") {
+  var x = ${Type}.min
+  x &-= 1
+  expectEqual(x, ${Type}.max)
+}
+
+tests.test("${Type}/MaskingMultiply") {
+  var x = ${Type}.max
+  x &*= 2
+  expectEqual(x, ${Type}.max &* 2)
```

Comment:
Since these operators are defined in terms of one another, it makes the test ... not testing anything =)
How about something like `expectEquals(${Type}.min, (${Type}.max / 2 + 1) &* 2)`?

Reply:
Good catch, I couldn't think of a guaranteed single way to test multiplication overflow across all types 🙂 

What about this?
```
tests.test("${Type}/Multiply/MaskingOverflow") {
  expectEqual((${Type}.max / 2 + 1) &* 2, ${Type}.min)
}

tests.test("${Type}/MultiplyInPlace/MaskingOverflow") {
  var x = ${Type}.max / 2 + 1
  x &*= 2
  expectEqual(x, ${Type}.min)
}
```

Reply:
in `expectXXX` functions, the expected value is the first parameter.

Reply:
Got it.